src/site/pages/digital/tsconfig.json

```diff
@@ -12,6 +12,7 @@
             "shared/*":       ["../../shared/*"],
         }
     },
-    "include": ["../../shared/utils/types"],
+    // TODO[model_refactor_api](Leon) - Add back "src" to include
+    "include": ["src/utils/ExpressionParser", "../../shared/utils/types", "tests/utils/ExpressionParser.test.ts"],
```

Comment:
Why do you need to add these files?

Reply:
So test can be run. It will have to be changed anyways before the main model refactor pr is merged so I just left it as is